test/index.ts

```diff
@@ -1,25 +1,59 @@
-import { fastify } from 'fastify';
-import { runServer } from '../src/server';
-import dotenv from 'dotenv';
 import axios from 'axios';
+import * as chai from 'chai';
```

Comment:
Acho que compensa mais vc importar direto o expect:

```suggestion
import { expect } from 'chai';
```